core-web/libs/edit-content/src/lib/components/dot-edit-content-field/dot-edit-content-field.component.html

```diff
@@ -1,87 +1,92 @@
 <label
     [attr.data-testId]="'label-' + field.variable"
-    [for]="field.variable"
     [checkIsRequiredControl]="field.variable"
-    dotFieldRequired
-    >{{ field.name }}</label
->
+    [for]="field.variable"
```

Comment:
the formater was changed? I see a lot of formater changes

Reply:
We have a lint for alphabetical order `"prettier-plugin-organize-attributes": "^0.0.5",` in our `package.json` that is the reason of why was moved. The other question is, why no was ordered when other dev touched the file.

Reply:
I created an issue for gardening day to take a look deep into that.